### ISSUE TITLE
OMIM "included" entries and not the preferred names on MONDO terms

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#2808]
NV to do:  _updated 02/10/22_
I need to write out concrete action items for Nico to work on this.

**Comment by kanems** [comment]
I have run into a couple of cases where an OMIM entry is only once in MONDO, but the disease name it is associated with is the OMIM "included" disorder, not the preferred name from OMIM. We handle these "included" terms as distinct entries in MedGen and thus cannot link a MONDO ID to the term which is OMIM's preferred term. Some examples:
I.	MONDO:0007691 “chronic inflammatory demyelinating polyneuropathy”
     a.	Maps to OMIM https://www.omim.org/entry/139393 but this represents Guillain-Barre syndrome, Familial AND “other entity” POLYNEUROPATHY, INFLAMMATORY DEMYELINATING, CHRONIC, INCLUDED; CIDP, INCLUDED
     b.	UMLS splits the MIM term among multiple CUIs; we split the OMIM entry into distinct concepts for GBS, familial and then imported the MONDO term separately.  However put the MIM# on the entry with the preferred name (In this case would be Guillain-Barre syndr, familial.)
      
II.	MONDO:0007608 Desmoid tumor
     a.	MIM https://omim.org/entry/135290 is for both desmoid disease, hereditary. AND “other” Desmoid Tumor (caused by somatic mutation)
     b.	MONDO includes the Fibromatosis, familial infiltrative (FIF) alt name, but not desmoid disease, hereditary. So we can’t have a MONDO ID for our entry that represents the PT from OMIM (DESMOID DISEASE, HEREDITARY) UMLS CUI: C1851124 
    c. is this one of the instances where maybe the disorders are similar enough? I'm not a clinician so I can't say.

Nicole, I see your ticket #2745, so it looks like there is a mechanism to manage OMIM entries relating to multiple disease names, I hope that means MONDO can link the MIM#s to the terms of the preferred name and (as needed) the "included" terms.

**Comment by sabrinatoro** [comment]
Hello @kanems 
I'm Sabrina, a new curator who recently joined the Mondo team.
We will address your issue as soon as possible. Thank you for your patience.
If this is a high priority, please let me know and we can prioritize it. Thank you!

**Comment by nicolevasilevsky** [comment]
I asked Ada about this and this was her response:
In most cases, an included title is not the same concept as the parent entry, so the UMLS has handled this correctly.  If you are linking to OMIM so that people can read about it, then use the parent MIM number, but ontologically it cannot be represented as the same.  This is the opposite from alternate titles which are synonyms.  Sometimes included concepts are off-shoots or not clearly distinct.  We may not have looked at them for some time, as well.

**Comment by kanems** [comment]
Hi Nicole, Ada's response is how I understood the distinction between "included" vs "alternate" terms on the same OMIM number, which is reassuring. However, my question to Mondo may not have been clear initially, let's see if I can clarify. 
Is Mondo's policy to map OMIM numbers based on the main "preferred" name on the record or on the "included" name? 
MedGen maps on the "preferred name" from OMIM and so if Mondo does the opposite, we cannot replicate the Mondo:OMIM relationship in MedGen, but we'd like to agree with Mondo as much as possible (we're just agreeable folks like that ;-) )
One example is OMIM 139393, which, as I read it, has the *primary* preferred name of "GUILLAIN-BARRE SYNDROME, FAMILIAL" (see screen shot). However, this OMIM number maps to MONDO:0007691 "chronic inflammatory demyelinating polyneuropathy" which has a related synonym "Guillain-Barre syndrome, familial."  So I think Mondo has appropriately handled that these two disease terms are related but not exactly the same, but I was confused by the selection to use the "included" disease rather than the OMIM primary name on the number.
![image](https://user-images.githubusercontent.com/71083858/129367220-7294929d-2830-4e6d-aead-bf397965ee6c.png)

**Comment by nicolevasilevsky** [comment]
Depends on @jiaola's work on the OMIM ingest.

@kanems - I'll respond to your comment above soon, thanks!

**Comment by nicolevasilevsky** [comment]
to do:
Would be nice to have a new xref tag, something like:
MONDO:includedInOMIM or
MONDO:splitFromOMIM

**Comment by nicolevasilevsky** [comment]
@kanems 
we renamed MONDO:0007691 to 'Guillain-Barre syndrome, familial', it should be available in the next release (early September)

We want to use the OMIM preferred labels for Mondo terms, we'll work on cleaning this up. Based on Ada's response too, we'll work on retroactively splitting out the terms that are 'included' in OMIM entries, as these should be separate classes, as noted above.

Thanks for bringing this to our attention! It will take some time as our developer Dazhi is working on improving the OMIM ingest workflow.

**Comment by sabrinatoro** [comment]
On QC call (211105):
Need to break down into a step by step to-do list.
During OMIM ingest, we should tag synonyms to indicate the "included" terms, so curators can go back later and decide whether these "included terms" should be synonyms or a new term. 
This requires a lot of work in the ingest pipeline. Will be done in 2022.

**Comment by maglott** [comment]
NCBI and UMLS have used the strategy of treating each 'included' term as a distinct concept and merging later if review indicates that some of the included terms within a MIM record are synonyms to each other. It is unlikely that an included term is a synonym to the concept represented by the MIM number title and alternate terms, so it may be appropriate to treat as distinct until proven otherwise , rather than treating as the same and then having to split.  
caveat:  I am using synonym here as 'different terms for the exact same concept'

**Comment by nicolevasilevsky** [comment]
I agree with @maglott's suggestion. I think it would be best if we added 'included' entries as new terms in Mondo and they would have the xref to the OMIM ID with a new tag MONDO:includedInOMIM.

I'm not sure how difficult it would be to do this retroactively?
